src/legacy/server/config/schema.js

```diff
@@ -231,6 +231,15 @@ export default () =>
       locale: Joi.string().default('en'),
     }).default(),
 
+    // temporarily moved here from the (now deleted) kibana legacy plugin
+    kibana: Joi.object({
+      enabled: Joi.boolean().default(true),
+      index: Joi.string().default('.kibana'),
+      autocompleteTerminateAfter: Joi.number().integer().min(1).default(100000),
+      // TODO Also allow units here like in elasticsearch config once this is moved to the new platform
+      autocompleteTimeout: Joi.number().integer().min(1).default(1000),
+    }).default(),
+
```

Comment:
Some legacy code are still accessing these config properties from legacy. I moved them into the server's schema for now. This will have to be handled properly when we remove the legacy server totally.

Reply:
We can remove quite a few validations here, not in the current PR, for sure